core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/annotations/Plugin.kt

```diff
@@ -43,3 +43,6 @@ public annotation class ScopeProperty
 
 @Target(AnnotationTarget.FUNCTION)
 internal annotation class Check
+
+@Target(AnnotationTarget.FUNCTION)
+internal annotation class ApiOverload
```

Comment:
Please add a small KDoc how it should be used and what it does.

Also, if we would go with `dataframe-core-minimized`, we could make this class be named `@ExtendedApi` or something. Indicating that this API will be larger.

Makes it also easier to recommend to users in the future: "If you don't have or you cannot run the compiler plugin, you can use the extended API to make use of the old column accessors or KProperties"

Reply:
I'll provide explanation here